testing/constraints-3.10.txt

```diff
@@ -2,5 +2,4 @@
 httplib2==0.19.0
 google-auth==1.32.0
 google-auth-httplib2==0.2.0
-google-api-core
```

Comment:
For prototyping purposes only, revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI